packages/hub/src/lib/create-repo.ts

```diff
@@ -54,4 +54,6 @@ export async function createRepo(params: {
 	if (!res.ok) {
 		throw await createApiError(res);
 	}
+	const output = await res.json();
+	return output.url;
```

Comment:
```suggestion
	return { repoUrl: output.url };
```

Maybe? To be consistent with `commit` that returns `{ pullRequestUrl: string, ...}`

Reply:
Yes, why not.

More generally i would try to be consistent with `huggingface_hub` whenever we can (cc also @Wauplin)